pom.xml

```diff
@@ -105,8 +105,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
seems fine but unrelated to this change? did your change necessitate this, or were you unable to run the tests even on master?

Reply:
Yeah I couldn't run tests on master either got this error:
<img width="566" alt="Screen Shot 2019-07-02 at 2 47 03 PM" src="https://user-images.githubusercontent.com/16430092/60548952-4837db80-9cd8-11e9-81f7-d79876dbc374.png">

Reply:
I'm going to break this out into another PR